utils/file/file.go

```diff
@@ -106,7 +110,16 @@ func GetYAML(taskInfo *mesos.TaskInfo) []string {
 // In case they have different depth of folders to keep compose files, GetDirFilesRecv help to get the complete path of
 // compose file
 func GetDirFilesRecv(dir string, files *[]string) {
-	if d, _ := os.Stat(dir); !d.IsDir() && (strings.HasSuffix(dir, ".yml") || strings.HasSuffix(dir, ".yaml") || dir == "yaml") {
+	d, err := os.Stat(dir)
+	if err != nil {
+		log.Errorf("Error getting file info of %s:%v\n", dir, err)
+		return
+	}
+	if d == nil {
```

Comment:
Actually if `err == nil`, `d` cannot be nil, so this check is not needed...

https://golang.org/src/os/stat_unix.go